PENDING.md

```diff
@@ -68,3 +69,14 @@ BUG FIXES
 *  \#1828 Force user to specify amount on create-validator command by removing default
 *  \#1839 Fixed bug where intra-tx counter wasn't set correctly for genesis validators
 * [tests] \#1675 Fix non-deterministic `test_cover` 
+* [client] \#1551: Refactored `CoreContext`
+  * Renamed `CoreContext` to `QueryContext`
```

Comment:
propose: QueryContext -> CliContext.  Since Query isn't the only thing we're doing there.